web/modules/custom/joinup_core/joinup_core.install

```diff
@@ -60,3 +62,94 @@ function joinup_core_requirements($phase) {
 function joinup_core_update_8100(&$sandbox) {
   \Drupal::service('module_installer')->install(['config_sync']);
 }
+
+/**
+ * Update the type of collections, solutions, releases and distributions.
+ *
+ * Two things will happen here.
```

Comment:
This block of text should be removed from the docblock. The reason is that the docblock description is outputted in in the updates list (see the screenshot) and it might confuse the user/admin. Instead you can move it as inline comment inside the function. Also, I think the 2 "things" deserve to be numbered or bulleted.
![1__php](https://user-images.githubusercontent.com/473868/35964179-9af13312-0cbf-11e8-86cf-a65062c624e3.jpg)

Reply:
Opps.. I thought only the first line was presented there.